scripts/qa_crowbarsetup.sh

```diff
@@ -1596,6 +1553,11 @@ EOF
     zypper -n dup -r Cloud -r cloudtup || zypper -n dup -r Cloud
     zypper rl kernel-default
 
+    # Workaround chef-solr crashes
+    if [ "$(uname -m)" = "aarch64" ]; then
+        ensure_packages_installed java-1_7_0-openjdk java-1_7_0-openjdk-headless
+    fi
```

Comment:
Probably yes, I'll fine one when I get around it (currently trying to fix something else)